modules/ossm-supported-configurations.adoc

```diff
@@ -11,6 +11,11 @@ The following configurations are supported for the current release of  {ProductN
 
 * Red Hat {product-title} version 4.x.
 
+[NOTE]
+====
+OpenShift Online is not supported for {ProductName}.
```

Comment:
I feel obliged to suggest "Red Hat OpenShift Online" unless that appears earlier in the assembly.